src/app/core/services/user/user.service.ts

```diff
@@ -23,6 +26,7 @@ export class UserService {
       .getLocalData(localKeys.USER_DETAILS)
       .then((data: any) => {
         this.userDetail=data;
+        this.userEvent.next(data.user);
```

Comment:
Why do we need this here?

Reply:
after login close app and open we need to to set the user data to the subject to show in side menu